.github/workflows/ci-linkchecks.yml

```diff
@@ -13,20 +13,13 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: "Generate token"
-        uses: actions/create-github-app-token@v2
-        id: generate-token
-        with:
-          app-id: ${{ secrets.AUTH_APP_ID }}
-          private-key: ${{ secrets.AUTH_APP_PRIVATE_KEY }}
-
       - name: Link Checker
         id: lychee
         uses: lycheeverse/lychee-action@1d97d84f0bc547f7b25f4c2170d87d810dc2fb2c
         with:
-          token: ${{ steps.generate-token.outputs.token }}
```

Comment:
This was a slightly desperate attempt to sledgehammer the problems out of the way - the `scitools-ci` app has highly elevated permissions!

I have since read the documentation and it seems that ANY token ought to be enough to give us access to the higher tier of allowed GitHub requests.

https://lychee.cli.rs/troubleshooting/rate-limits/#github-rate-limiting